src/earn/EarnApyAndAmount.tsx

```diff
@@ -24,14 +22,12 @@ export function EarnApyAndAmount({
 }) {
   const { t } = useTranslation()
 
-  const localCurrencySymbol = useSelector(getLocalCurrencySymbol) ?? LocalCurrencySymbol.USD
-
   const asyncPoolInfo = useAavePoolInfo({ depositTokenId: token.tokenId })
   const apy = asyncPoolInfo?.result?.apy
 
   const apyString = apy ? (apy * 100).toFixed(2) : '--'
   const earnUpTo =
-    apy && tokenAmount?.gt(0) ? tokenAmount.multipliedBy(new BigNumber(apy)).toFormat(2) : '--'
+    apy && tokenAmount?.gt(0) ? tokenAmount.multipliedBy(new BigNumber(apy)) : new BigNumber(0)
```

Comment:
When APY is loading this will now show earn up to 0 / yr instead of -- / yr, right? Just confirming that this is the behavior we want for the loading state. It seems like it would be tricky to have it show something different while using TokenDisplay

Reply:
Actually, is there a reason the skeleton placeholder was removed?

Reply:
That's correct, it'll display `0 / yr` instead of `-- / yr` as the `TokenDisplay` component does not contain a loading state.